README.md

```diff
@@ -160,15 +160,16 @@ cd $(mktemp -d ${TMPDIR:-/tmp}/repro-XXXXXXX)
 datalad create -d ds000003-qc -c text2git
 cd ds000003-qc
 # Install our containers collection:
-datalad install -d . ///repronim/containers
+datalad install -d . -s ///repronim/containers code/containers
 # Optionally -- copy container of interest definition to the current (or desired)
 # version # to facilitate reproducibility while still being able to upgrade containers
 # subdataset if so desired to get access to newer versions.
 # We will also use 0.16.0 since newer ones require more memory and
 # would fail to run on CI.
-containers/scripts/freeze_versions --save-dataset=. bids-mriqc=0.16.0
 # Install input data:
 datalad install -d . -s https://github.com/ReproNim/ds000003-demo sourcedata
+datalad run -m "Downgrade/Freeze mriqc container version" \
+  code/containers/scripts/freeze_versions --save-dataset=. bids-mriqc=0.16.0
```

Comment:
why this reordering was necessary?

note that it as well left comment above !

Reply:
I put it back by hand and "missed". Fixed in latest push.